core/chain/src/mempool.rs

```diff
@@ -0,0 +1,112 @@
+use std::collections::HashSet;
```

Comment:
hm, why not make a mempool it's own cargo?

Reply:
Good point. I was lazy :)